ReleaseNotes.md

```diff
@@ -2,7 +2,8 @@
 
 - Update function to enable background location update when app is in While in use location permissions.
 - Upgraded to react 18.0.0 and react-native 0.69.4
-- Updated to Point Android SDK v15.4.2
-- Updated to Point iOS SDK v15.6.4
+- Updated to Point Android SDK v15.5.1
+- Updated to Point iOS SDK v15.6.5
 - Fix `"new NativeEventEmitter()" was called with a non-null argument without the required "some" method` warnings which appear with using react-native versions `0.65.0` or higher.
 - Fix issue when OnZoneInfoUpdate return empty ZoneInfo list
+- Remove Javadoc to fix expo run issue.
```

Comment:
I think we can remove the other lines?

Reply:
Probably just 

> Upgraded to react 18.0.0 and react-native 0.69.4

and

> Fix `"new NativeEventEmitter()" was called with a non-null argument without the required "some" method` warnings which appear with using react-native versions `0.65.0` or higher.

?

Reply:
I thought the only changes in 2.2.1 would be below? We don't need to mention remove javadoc as it has no impact to the customers.
```
- Updated to Point Android SDK v15.5.1
- Updated to Point iOS SDK v15.6.5
```

Reply:
The bluebar feature is also new in 2.2.1 ?
> Update function to enable background location update when app is in While in use location permissions.

Reply:
ah yep. 👍

Reply:
Updated